### ISSUE TITLE
CLI: cannot read 'extraneous'

### ISSUE THREAD

**Issue by jereckley** [kamilkisiela/graphql-hive#633]
Cannot install latest

Replicate with a simple ` npm install @graphql-hive/cli -g`

```
npm ERR! Cannot read properties of undefined (reading 'extraneous')

```

**Comment by kamilkisiela** [comment]
https://github.com/npm/cli/issues/4290

**Comment by kamilkisiela** [comment]
It's super weird it happens during installation and only on NPM, works perfectly fine with Yarn.

**Comment by kamilkisiela** [comment]
Fixed in `0.19.8`
